cloud dm: update docs as per new UI (#13860)
pingcap/docs: tidb-cloud/import-parquet-files.md

3. Provide the following information for the source Parquet files:

    - **Location**: select **Amazon S3**.
    - **Data format**: select **Parquet**.
    - **Bucket URI**: select the bucket URI where your Parquet files are located.
    - **Bucket Access** (This field is visible only for AWS S3): you can use either an AWS access key or a Role ARN to access your bucket. For more information, see [Configure Amazon S3 access](/tidb-cloud/config-s3-and-gcs-access.md#configure-amazon-s3-access).
        - **AWS Access Keys**: enter the AWS access key ID and AWS secret access key.
        - **AWS Role ARN**: enter the Role ARN value.

4. You can choose to **Import into Pre-created Tables**, or **Import Schema and Data from S3**.

> 5. If you want to modify the file patterns, click **Advanced Settings** and then click **Mapping Settings**.

Reviewer: ```suggestion
5. If you want to import the source files that don't meet the naming conventions, click **Advanced Settings** and then click **Mapping Settings**.
```

5. If the source files do not meet the naming conventions, you can specify a custom mapping rule between a single target table and the CSV file. After that, the data source files will be re-scanned using the provided custom mapping rule. To modify the mapping, click **Advanced Settings** and then click **Mapping Settings**. Note that **Mapping Settings** is available only when you choose **Import into Pre-created Tables**.

    You can also use wildcards to match the source files. For example:

    - `s3://<bucket-name>/path/my-data?.parquet`: all Parquet files starting with `my-data` and one character (such as `my-data1.parquet` and `my-data2.parquet`) in that folder will be imported into the same target table.
    - `s3://<bucket-name>/path/my-data*.parquet`: all Parquet files in the folder starting with `my-data` will be imported into the same target table.

    Note that only `?` and `*` are supported.

    - **Target Database**: enter the name of the target database you select.

    - **Target Tables**: enter the name of the target table you select. Note that this field only accepts one specific table name, so wildcards are not supported.

>     - **Source file URIs and names**: enter the source file URI and name, for example, `s3://sampledate/ingest/TableName.01.parquet`. Note that the folder name of the URI must contain the data source folder. For more information about the URI format, see [URI format](https://docs.pingcap.com/tidb/stable/backup-and-restore-storages#uri-format).

Author: ```suggestion
    - **Source file URIs and names**: enter the source file URI and name in the following format `s3://[bucket_name]/[data_source_folder]/[file_name].parquet`. For example, `s3://sampledate/ingest/TableName.01.parquet`. 

    > **Note:**
    >
    > The URI must contain the data source folder.
    
```

    - **Bucket URI**: select the bucket URI where your Parquet files are located.
    - **Bucket Access** (This field is visible only for AWS S3): you can use either an AWS access key or a Role ARN to access your bucket. For more information, see [Configure Amazon S3 access](/tidb-cloud/config-s3-and-gcs-access.md#configure-amazon-s3-access).
        - **AWS Access Keys**: enter the AWS access key ID and AWS secret access key.
        - **AWS Role ARN**: enter the Role ARN value.

4. You can choose to **Import into Pre-created Tables**, or **Import Schema and Data from S3**.

    - **Import into Pre-created Tables** allows you to create tables in TiDB in advance and select the tables that you want to import data into. In this case, you can choose up to 1000 tables to import. You can click **Chat2Qury** in the left navigation pane to create tables. For more information about how to use Chat2Qury, see [Explore Your Data with AI-Powered Chat2Query](/tidb-cloud/explore-data-with-chat2query.md).
    - **Import Schema and Data from S3** allows you to import SQL scripts for creating a table and import corresponding table data stored in S3 into TiDB.

> 5. If the source files do not meet the naming conventions, you can specify a custom mapping rule between a single target table and the CSV file. After that, the data source files will be re-scanned using the provided custom mapping rule. To modify the mapping, click **Advanced Settings** and then click **Mapping Settings**. Note that **Mapping Settings** is available only when you choose **Import into Pre-created Tables**.

Reviewer: ```suggestion
5. If the source files do not meet the naming conventions, you can specify a custom mapping rule between a single target table and the CSV/Parquet/SQL file. After that, the data source files will be re-scanned using the provided custom mapping rule. To modify the mapping, click **Advanced Settings** and then click **Mapping Settings**. Note that **Mapping Settings** is available only when you choose **Import into Pre-created Tables**.
```
